app/scripts/modules/tencentcloud/src/tencentcloud.setting.ts

```diff
@@ -0,0 +1,35 @@
+import { IProviderSettings, SETTINGS } from '@spinnaker/core';
+
+export interface IClassicLaunchWhitelist {
+  region: string;
+  credentials: string;
+}
+
+export interface ITencentCloudProviderSettings extends IProviderSettings {
+  defaults: {
+    account?: string;
+    region?: string;
+    subnetType?: string;
+    vpc?: string;
+  };
+  defaultSecurityGroups?: string[];
+  loadBalancers?: {
+    inferInternalFlagFromSubnet: boolean;
+    certificateTypes?: string[];
+  };
+  useAmiBlockDeviceMappings?: boolean;
+  classicLaunchLockout?: number;
+  classicLaunchWhitelist?: IClassicLaunchWhitelist[];
```

Comment:
Does tencent really have AmiBlockDeviceMappings or ClassicLaunch?  Both of these things imply Amazon to me.

Reply:
> I found a couple references to other cloud providers that look possibly misplaced.

Thanks for your earnest review, I fixed them